mava/configs/env/lbf.yaml

```diff
@@ -4,7 +4,8 @@ defaults:
   - scenario: 2s-8x8-2p-2f-coop # [2s-8x8-2p-2f-coop, 8x8-2p-2f-coop, 2s-10x10-3p-3f, 10x10-3p-3f, 15x15-4p-3f, 15x15-4p-5f]
 # Further environment config details in "2s-8x8-2p-2f-coop" file.
 
-env_name: LevelBasedForaging-v0
+env_name: LevelBasedForaging
```

Comment:
Can we please just add a comment in the env yaml files that the env_name is just for logging purposes.